pages/en/lb4/DB2-for-i.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "DB2 for i connector"
+lang: en
+layout: readme
+source: loopback-connector-ibmi
+keywords: LoopBack
+tags: [connectors, readme]
+sidebar: lb4_sidebar
```

Comment:
you may want to add this item on the sidebar page as well :p

Reply:
Yes, i'm working (picking up) on this PR: https://github.com/strongloop/loopback-next/pull/4409.

Reply:
it might be good to make sure https://github.com/strongloop/loopback-next/pull/4409 got landed before this lands. Thanks for the reminder!